modules/plugin/imagemosaic/src/main/java/org/geotools/gce/imagemosaic/GranuleDescriptor.java

```diff
@@ -1735,8 +1711,8 @@ GranuleOverviewLevelDescriptor getLevel(final int index) {
                 throw new IllegalArgumentException(
                         "Unable to get an ImageReader for the provided file "
                                 + granuleUrl.toString());
-            final boolean ignoreMetadata = customizeReaderInitialization(reader, null);
-            reader.setInput(inStream, false, ignoreMetadata);
+
+            reader.setInput(inStream, false, false);
```

Comment:
Hum... why is it fixed to false here, but a similar call use ignoreMetadata before in the same file?

Reply:
an oversight while refactoring the initialization. will fix it. thx

Reply:
Done